bower.json

```diff
@@ -20,7 +20,8 @@
   "homepage": "https://github.com/AqoviaElements/epi-cms-property",
   "ignore": [],  
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.4.0"
+    "polymer": "Polymer/polymer#^1.4.0",
+    "properties-from-ancestor-behavior": "~1.0"
```

Comment:
Yes. I'm treating the decision that we recently agreed for the internal `inventory-overview` package (to start using SemVer with `~`), as applying to the team that authors the package.  

My idea is to change it to ^ once we have evidence that most of team has been marking breaks correctly.

Don't mind if you'd rather start trustful. I can change it.